pages/_app.tsx

```diff
@@ -15,14 +15,17 @@ export default function App(props: AppProps) {
   return (
     <RecoilRoot>
       <Head>
-        <title>FQM Testify: an ECQM Analysis Tool</title>
+        <title>FQM Testify: an eCQM Analysis Tool</title>
```

Comment:
```suggestion
        <title>FQM Testify: a FHIR-based eCQM Analysis Tool</title>
```
Might be good to add this so that it aligns with the README

Reply:
Nice!